docs/pages/reference/agent-services/database-access-reference/configuration.mdx

```diff
@@ -181,6 +181,13 @@ spec:
     # connection made by a user.
     server_version: 8.0.28
 
+  # Oracle only options.
```

Comment:
sadly, there is one more file to update:
https://github.com/gravitational/teleport/blob/master/docs/pages/includes/config-reference/database-config.yaml